library/src/main/java/com/datatorrent/lib/state/managed/AbstractManagedStateImpl.java

```diff
@@ -226,6 +232,15 @@ protected int prepareBucket(long bucketId)
     return bucketIdx;
   }
 
+  protected void putInBucket(long bucketId, long timeBucket, Slice key, Slice value)
+  {
+    if (timeBucket != -1) {
+      //time bucket is invalid data is not stored
```

Comment:
Suggest moving comment above the if and rephrasing:
// if time bucket is invalid, data is not stored